feat(docs): direct minting parameters and settings (#1340)
flare-foundation/developer-hub: docs/fassets/reference/IMintingTagManager.mdx

---
title: IMintingTagManager
description: FAssets IMintingTagManager interface reference.
keywords: [fassets, minting-tags, xrp, bitcoin, dogecoin, flare-network]
sidebar_position: 5
---

Reference for managing and interacting with FAssets `IMintingTagManager`.

> To get the minting tag manager address, use the [getMintingTagManager](/fassets/reference/IAssetManager#getmintingtagmanager) method on the [IAssetManager](/fassets/reference/IAssetManager) contract.

Reviewer: ```suggestion
To get the minting tag manager address, use the [getMintingTagManager](/fassets/reference/IAssetManager#getmintingtagmanager) function of the [IAssetManager](/fassets/reference/IAssetManager) contract.
```

## Functions

### `reserve`

Reserve a new minting tag by paying the reservation fee.
The caller becomes the owner of the tag and the initial minting recipient.

> 

Reviewer: Is this the right place to mention that the tag is an NFT?

Author: Improved the text
